Add salvage.mdx (#1658)
utelecon/utelecon.github.io: src/pages/en/systems/leave/salvage.mdx

---
title: "Trial Operation: Cloud Data Salvage after UTokyo Account Revocation"
breadcrumb:
  title: Cloud Data Salvage
---
<div class="box">

>     The "Cloud Data Salvage" described on this page (a special measure to temporarily provide access to your data and accounts from your period of enrollment after your UTokyo Account has been revoked) will be available for application from **October 1, 2025**. Pre-linking with the “[UTokyo Account Identity Verification Service](/en/utokyo_account/ident-myna/)” will be available from the evening of September 10. Please ensure you complete the pre-linking process **before you leave the university**.

Reviewer: - capitalizeしなくてよいのでは（ほかの箇所も同様）
```suggestion
    The "cloud data salvage" described on this page (a special measure to temporarily provide access to your data or account from your period of enrollment after your UTokyo Account has been revoked) will be available for application from **October 1, 2025**. Pre-linking in the “[UTokyo Account Identity Verification Service](/en/utokyo_account/ident-myna/)” will be available from the evening of September 10. Please ensure you complete the pre-linking process **before you leave the university**.
```

</div>

When you leave the University of Tokyo upon graduation, resignation, etc., your UTokyo Account will be revoked at the end of your period of enrollment, and you will no longer be able to sign in. Therefore, if you wish to continue using data stored in various information systems such as Zoom, ECCS Cloud Email, and Microsoft 365, you must back up your data by following the procedures on the "[Backing Up Your Data When Leaving the University](./backup/)" page.

> However, if for some reason you are unable to create a backup before leaving and you have a legitimate reason to access your data or accounts after you have left, we may be able to provide exceptional access to some of your data and accounts under certain conditions. This is a special measure called "Cloud Data Salvage". By using Cloud Data Salvage, you can temporarily back up your ECCS Cloud Email history and data on Google Drive/OneDrive, or add email forwarding settings to your ECCS Cloud Email.

Reviewer: - justifiableの方がニュアンスがありそう？　画面でもjustifiableと書いているのでこちらでお願いします。
- この場合アカウントは単数でよさそうです（ほかの箇所も同様）
```suggestion
However, if for some reason you are unable to create a backup before leaving and you have a justifiable reason to access your data or account after you have left, we may be able to provide exceptional access to some of your data and accounts under certain conditions. This is a special measure called "Cloud Data Salvage". By using Cloud Data Salvage, you can temporarily back up your ECCS Cloud Email history and data on Google Drive/OneDrive, or add email forwarding settings to your ECCS Cloud Email.
```

## Conditions for Using Cloud Data Salvage

There are several conditions for using Cloud Data Salvage. Please understand the following conditions before applying.

* **Who can apply**: Former members of the University of Tokyo who have left due to graduation, resignation, etc., and are **within 180 days of their UTokyo Account being revoked**.
  * Please note that even within 180 days of account revocation, some or all of your data may be permanently deleted and become unavailable due to circumstances such as overall university storage capacity. Please be aware that **the fundamental principle is [to back up your data before you leave](./backup/)**.
* **Eligible Services**: Among the cloud services available with a UTokyo Account, the following are eligible.
  * **[ECCS Cloud Email](/en/google/) account**
    * We will enable you to log in to your ECCS Cloud Email account with a special password, without going through your UTokyo Account.
    * This will give you access to all data within the Google system, such as Gmail and Google Drive.
    * You will also be able to perform various operations within your Google account, such as adding a forwarding setting in Gmail.

>     * However, for email addresses from [the Mail Hosting Service](https://mh.ecc.u-tokyo.ac.jp/) (email addresses with domains other than g.ecc.u-tokyo.ac.jp that can be used with ECCS Cloud Email), please consult the respective domain administrator separately.

Reviewer: ```suggestion
    * However, for email addresses from [the Mail Hosting Service](https://mh.ecc.u-tokyo.ac.jp/) (only in Japanese) (email addresses with domains other than `g.ecc.u-tokyo.ac.jp` that can be used with ECCS Cloud Email), please consult the respective domain administrator separately.
```

If you are unable to use the UTokyo Account Identity Verification Service, please see "[Procedure for When the UTokyo Account Identity Verification Service is Unavailable](#no-myna)".

// Instructions on the procedure will be added here.

You will be notified of the approval or rejection of your Cloud Data Salvage application by email. If your application is approved, please back up your data, set up forwarding, etc., within one week, following the instructions on the "[Backing Up Your Data When Leaving the University](./backup/)" page.

### Procedure for When the UTokyo Account Identity Verification Service is Unavailable
{:#no-myna}

As a general rule, the "UTokyo Account Identity Verification Service" is used for identity verification for Cloud Data Salvage. However, if you are unable to use this service, we will accept your application on an exceptional basis provided that you meet both of the following conditions:

* You were enrolled at the University of Tokyo as a student immediately before leaving.
* You have consulted with the academic affairs office of your former college/faculty/graduate school and have obtained their cooperation for identity verification.

> If you wish to use the Cloud Data Salvage service, please contact the academic affairs office of the college/faculty/graduate school you belonged to immediately before leaving. Inform them that "you would like to retrieve data from your UTokyo Account and request identity verification and forwarding of your request to the utelecon support desk," and provide them with the following required information.

Reviewer: - 間接話法（接続詞thatで繋ぐ、代名詞はyou）のとき引用符は使っていいんでしたっけ？（自信なし）
- informも微妙にニュアンスがあってるか自信がないです
```suggestion
If you wish to use the Cloud Data Salvage service, please contact the academic affairs office of the college/faculty/graduate school you belonged to immediately before leaving. Inform them that "you would like to retrieve data from your UTokyo Account and request identity verification and forwarding of your request to the utelecon support desk," and provide them with the following required information.
```
